src/slave/containerizer/mesos/isolators/xfs/disk.cpp

```diff
@@ -274,6 +274,8 @@ Future<Nothing> XfsDiskIsolatorProcess::recover(
     alive.insert(state.container_id());
   }
 
+  list<string> unmanaged;
```

Comment:
we default to vector unless list is needed for e.g. O(1) insertion, so: s/list/vector/